testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCloneTableCompatibility.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.deltalake;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.testng.services.Flaky;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_EXCLUDE_73;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_EXCLUDE_91;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_OSS;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.dropDeltaTableWithRetry;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+
+public class TestDeltaLakeDatabricksCloneTableCompatibility
+        extends BaseTestDeltaLakeS3Storage
+{
+    public static final String VERSION_AS_OF_2 = " VERSION AS OF 2";
+    public static final String VERSION_AS_OF_1 = " VERSION AS OF 1";
+    public static final String SHALLOW_CLONE_DEFAULT = " SHALLOW CLONE default.";
+    public static final String TEST_DL_BASE_TABLE = "test_dl_base_table_";
+    public static final String CREATE_TABLE_DEFAULT = "CREATE TABLE default.";
+    public static final String INSERT_INTO_DEFAULT = "INSERT INTO default.";
+    public static final String SELECT_FROM_DEFAULT = "SELECT * FROM default.";
+    public static final String SELECT_FROM_DELTA_DEFAULT = "SELECT * FROM delta.default.";
+    public static final String DATABRICKS_COMPATIBILITY_TEST = "/databricks-compatibility-test-";
+    public static final String DEFAULT = "default.";
```

Comment:
Please inline these constants to improve readability.

Reply:
done